configs/datasets/vln/mp3d_r2r.yaml

```
@@ -0,0 +1,5 @@
+DATASET:
+  TYPE: R2RVLN-v1
+  SPLIT: train
+  DATA_PATH: "data/datasets/vln/r2r/v1/{split}/{split}.json.gz"
```

To be consistent with our dataset paths let change path to everywhere:
```suggestion
  DATA_PATH: "data/datasets/vln/mp3d/r2r/v1/{split}/{split}.json.gz"
```